lib/node/Helper.js

```diff
@@ -0,0 +1,78 @@
+const Negative = require('../Negative');
+const NodeCreator = require('./Creator');
+const NodeType = require('./Type');
+
+const NodeHelper = {};
```

Comment:
hmm trying to think of a more specific name than "helper" since that could be anything

NodeGeneralType? or maybe NodeCustomType?